<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>progetti_table</class>
 <widget class="QWidget" name="progetti_table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>813</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="button_aggiorna_progetto">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>760</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Aggiorna Progetto</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_progetti">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>971</width>
     <height>661</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>171</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Tipologia</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="combobox_tipologia_progetti"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_aggiungi_progetto">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>760</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Aggiungi Progetto</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>258</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Ricerca Progetti</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="search_progetti"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>171</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Stato</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="combobox_stato_progetti"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_dipendenti">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>760</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Dipendenti</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_crea_user">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>760</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Crea User</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_edit_user">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>760</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit User</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_logout">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>760</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>20</y>
     <width>171</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Ordinamento</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="combobox_ordinamento"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_azzera_filtri">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Azzera</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
